router/events.go

```diff
@@ -246,9 +245,12 @@ func (h *Handlers) HandleGetiCalByPrivateID(c echo.Context) error {
 	if err != nil {
 		return judgeErrorResponse(err)
 	}
-
-	cal := presentation.ICalFormat(events, h.Origin)
+	userMap, err := h.Repo.GetUserMap(info)
+	if err != nil {
+		return judgeErrorResponse(err)
+	}
+	cal := presentation.ICalFormat(events, h.Origin, userMap)
 	var buf bytes.Buffer
-	_ = ical.NewEncoder(&buf).Encode(cal)
+	cal.SerializeTo(&buf)
```

Comment:
ここlinterに怒られてるので変更前みたいに
```suggestion
	_ = cal.SerializeTo(&buf)
```
してほしいです:pray: (handlingしてもいいけど)